chromeBackground.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
Why have you added the chromeBackground page? Maybe is a rest of an older version? I think remember we removed them when we remove support for the Chrome OS.

Reply:
Agreed, there is no reason to re-add this.

Reply:
I had no clue either lol - it just appeared!  I'll get rid of it.  Thanks